Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM elixir:1.9.4-alpine
+
+MAINTAINER Stephen McCullough <stephen.mccullough@gmail.com> 
+
+RUN apk add postgresql-client
+RUN apk add nodejs
+RUN apk add --update npm
+
+
+ADD . /app
+WORKDIR /app
+
+RUN mkdir -p /app/assets
+WORKDIR /app
+
+ENV NODE_ENV=development
+
+COPY assets/package.json assets/package-lock.json ./assets/
+COPY deps/phoenix deps/phoenix
+COPY deps/phoenix_html deps/phoenix_html
+
+RUN mix local.hex --force 
```

Comment:
I think this should come before line 10 (which copies the code onto the image). This way, a code change doesn't bust the image cache.